core-kit/curated/sys-apps/baselayout/baselayout-3.0.ebuild

```diff
@@ -1,17 +1,20 @@
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
-inherit versionator prefix
+inherit prefix
 
 DESCRIPTION="Filesystem baselayout and init scripts"
-SRC_URI="https://gitweb.gentoo.org/proj/${PN}.git/snapshot/baselayout-2.6.tar.bz2"
+SRC_URI=""
```

Comment:
Empty SRC_URI?

Reply:
Yeah, we don't need anymore Gentoo. The sources are under FILESDIR/baselayout